src/main/java/org/csanchez/jenkins/plugins/kubernetes/volumes/workspace/DynamicPVCWorkspaceVolume.java

```diff
@@ -82,72 +63,17 @@ public void setStorageClassName(@CheckForNull String storageClassName) {
 
     @Override
     public Volume buildVolume(String volumeName, String podName) {
-        return new VolumeBuilder()
-                .withName(volumeName)
-                .withNewPersistentVolumeClaim()
-                .withClaimName("pvc-" + podName)
-                .withReadOnly(false)
-                .and()
-                .build();
+        return buildPVC(volumeName, podName);
     }
 
     @Override
-    public PersistentVolumeClaim createVolume(KubernetesClient client, ObjectMeta podMetaData){
-        String namespace = podMetaData.getNamespace();
-        String podId = podMetaData.getName();
-        LOGGER.log(Level.FINE, "Adding workspace volume from pod: {0}/{1}", new Object[] { namespace, podId });
-        OwnerReference ownerReference = new OwnerReferenceBuilder().
-                withApiVersion("v1").
-                withKind("Pod").
-                withBlockOwnerDeletion(true).
-                withController(true).
-                withName(podMetaData.getName()).
-                withUid(podMetaData.getUid()).build();
-
-         PersistentVolumeClaim pvc = new PersistentVolumeClaimBuilder()
-                .withNewMetadata()
-                .withName("pvc-" + podMetaData.getName())
-                .withOwnerReferences(ownerReference)
-                .withLabels(DEFAULT_POD_LABELS)
```

Comment:
Why removed the labels here? I used the labels to distinguish PVC created by Jenkins, and found it not worked after I upgraded the plugin to the latest version. :cry: